README.md

```diff
@@ -27,7 +26,7 @@ You need to set the `TRITON_PLUGINS_DIRS` environment variable to the location o
 export TRITON_PLUGIN_DIRS=$(pwd)/triton_shared
 
 git clone --recurse-submodules https://github.com/microsoft/triton-shared.git triton_shared
-cd triton_shared/triton
+cd triton_shared/triton/python
 ```
```

Comment:
A nice thing before was that the instructions were the same as OpenAI's for building Triton. If you make this change then you also have to change the command line to use current direct instead of the python directory:

pip3 install -e . --no-build-isolation